src/entry/params.c

```diff
@@ -68,6 +68,8 @@
             "prematurely after the test\n"                  \
     "    --verbose[=level]: sets verbosity to level "       \
             "(1 by default)\n"                              \
+    "    --crash: crashs failing tests so that an debugger" \
+            "can gain control\n"                            \
```

Comment:
Re-reading this, there are some errors: it should be "crash<e>s failing tests so that a debugger<SPACE>can gain control" (letters in brackets mine).

Maybe this could be reworded as "crash failing assertions rather than aborting (for debugging purposes)".